Makefile

```diff
@@ -1,41 +1,139 @@
 VERSION=$(shell git describe --tags --abbrev=0)
 COMMIT=$(shell git rev-parse --short HEAD)
+DOCKER_COMPOSE_FILE=./.devcontainer/docker-compose.yaml
 
 ifeq ($(VERSION),)
 	VERSION="0.0.0"
 endif
 
+# Run gocyclo to check the cyclomatic complexities of the top 10 most complex functions
 gocyclo:
 	go run github.com/fzipp/gocyclo/cmd/gocyclo@latest -top 10 -ignore "_test|contract_" ./
```

Comment:
There's a minor debt issue here. The make command should always execute the `build` target by default instead of the current `gocyclo`, a simple reordering of the targets can solve this issue.

https://www.gnu.org/software/make/manual/html_node/How-Make-Works.html

```
Node on  387-add-make-commands [$?] via 🐳 desktop-linux via 🐹 v1.22.1 via 🐍 v3.12.2 took 5s 
❯ make
go run github.com/fzipp/gocyclo/cmd/gocyclo@latest -top 10 -ignore "_test|contract_" ./
25 worker New internal/engine/worker/decentralized/factory.go:37:1
21 lens (*worker).Transform internal/engine/worker/decentralized/contract/lens/worker.go:122:1
21 ens (*worker).Transform internal/engine/worker/decentralized/contract/ens/worker.go:119:1
21 crossbell (*worker).Transform internal/engine/worker/decentralized/contract/crossbell/worker.go:124:1
19 ethereum (*worker).Transform internal/engine/worker/decentralized/core/ethereum/worker.go:110:1
19 uniswap (*worker).transformV3SwapLog internal/engine/worker/decentralized/contract/uniswap/worker.go:663:1
19 momoka (*worker).transformMomokaAction internal/engine/worker/decentralized/contract/momoka/worker.go:134:1
18 cockroachdb (*client).findIndexesPartitioned internal/database/dialer/cockroachdb/client_partitioned.go:400:1
17 monitor (*Monitor).flagWorkerStatus internal/node/monitor/monitor.go:152:1
17 oneinch (*worker).handleEthereumImplicitAggregationRouterTransaction internal/engine/worker/decentralized/contract/1inch/worker.go:162:1
```

Reply:
I just reordered the targets in Makefile to address this issue.